api/lagoon/client/_lgraphql/projects/projectGroups.graphql

```diff
@@ -0,0 +1,13 @@
+query (
+  $name: String!) {
+    projectByName(
+      name: $name) {
+        id
+        name
+        groups {
+          id
+          name
```

Comment:
if you can include `type` in the return too, that would be :chefkiss: that would be useful for other systems that may use this to do something with default groups which only exist with a `type`. The CLI could even add this as a field